File: pong/Assets/Scripts/Timer.js
#pragma strict

private var text : Text;       // Reference to the Text component.
var timer = 0.0;


function Awake ()
{
    text = GetComponent (Text);

}


function Update ()
{
    timer += Time.deltaTime;
    var intTime : int = timer;
    text.text = "Time: " + intTime;
}

File: pong/Assets/Scripts/Timer.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class Timer : MonoBehaviour
{
	float timer; 
	
	Text text; 
	
	
	void Awake ()
	{
		text = GetComponent <Text> ();
			}
	
	
	void Update ()
	{
		timer += Time.deltaTime;
		int intTime = (int) timer;
		text.text = "Time: " + intTime;
	}
}
